README.md

```diff
@@ -18,9 +18,9 @@ Add your answers to this markdown file.
 Three reasons what asymptotic analysis may be misleading with respect to actual performance in practice are hardware, constant factors, and run times. When an algorithm is ran on two seperate computers, if one computer has better hardware it will run faster. This can be misleading if the two computers are comparing times. This can be the case for two equal computers as well, just with slightly different parts, there is factors like the memory and CPU that could change results. An algorithm can ignore constant factors when using asymptotic analysis. Larger constant factors can make the algorithm slower for practical input sizes. Something else that can be misleading about asymptotic analysis is that two algorithms may have the same asymptotic complexity but have different run times. 
 
 
-The asymptotic time complexity of a binary serach tree is O(log n). The best way to find the run time will be to take the ratio of 1,000 elements and 10,000 elements. To do this we can do the following calculation, log2(10,000) = 13.29. We can use the value of 5 seconds for 1,000 elements to figure out the time for 10,000 elements. Do the calculation (13.29 * 5) and we get that the run time will be apporxiamtly 66.54 seconds. (I'm not sure if I am failing to understand this, or if I need to do log2(10,000)/log2(1000) and take that number times 5).
+The asymptotic time complexity of a binary serach tree is O(log n). The best way to find the run time will be to take the ratio of 1,000 elements and 10,000 elements. To do this we can do the following calculation, log2(10,000) = 13.29. Do the calculation (13.29 * 5) and we get that the run time will be apporxiamtly 66.54 seconds. 
```

Comment:
Wait, so the input is 10 times as much and you're getting more than 10 times the runtime? That would be worse than linear, certainly not logarithmic.